### PR TITLE
Nebula for autoversioning

### PR DESCRIPTION
**Pull request by mccorby**
This PR addresses the issue #15

### CODE REVIEW

**Comment by vkkhare** [comment]
looks great. Do we need to set up tags in the repo atm?

File: syftlib/publish.gradle [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
-    artifactId = artifactProjectID
-    publishVersion = publishVersionID
-    desc = 'Android worker library'
-    website = 'https://github.com/OpenMined/KotlinSyft'
-}
\ No newline at end of file
+def bintrayUser = properties.getProperty('bintray.user')
+def bintrayKey = properties.getProperty('bintray.key')
+
+bintray {
```
Comment (vkkhare) [inline syftlib/publish.gradle]: Awesone! Does the lib need tags to be added in the repo now? Additionally, if travis files need to be changed for knowing when to publish or use custom gradle build command?

Comment (mccorby) [reply syftlib/publish.gradle]: I'd go step by step. It already took me some time to figure out this first part :)
I think we need to set a first tag, yes. I will do that.
Then there is some other things to do if pushing from the command line with the branch name patterns but code-wise, it's ready.

Comment (vkkhare) [reply syftlib/publish.gradle]: Afaik you would need to change the Travis files as well. I would prefer that also gets added here in this PR

**Pull request merged**